Bookish/Bookish.Web/Views/Shared/_Layout.cshtml

```diff
@@ -26,13 +26,19 @@
                         {
                             <ul class="navbar-nav nav justify-content-between">
                                 <li class="nav-item">@Html.ActionLink("Home", "Index", "Home")</li>
-                                <li class="nav-item">@Html.ActionLink("Privacy", "Privacy", "Home")</li>
                                 <li class="nav-item">@Html.ActionLink("Library", "Library", "Home")</li>
-                            </ul>
-                        }
+                                <li class="nav-item">@Html.ActionLink("Add Book", "AddBook", "Home")</li>
+                                <li class="nav-item">@Html.ActionLink("Privacy", "Privacy", "Home")</li>
+                        </ul>
+                    }
+                    else
+                    {
+                    <ul>
+                        <li class="nav-item">@Html.ActionLink("Add Book", "AddBook", "Home")</li>
```

Comment:
This looks like it's saying you can add a book even if you aren't signed in - is that intentional?